crates/rmf_site_editor/src/occupancy.rs

```diff
@@ -143,6 +143,17 @@ pub struct CalculateGrid {
     pub ignore: HashSet<Entity>,
 }
 
+impl CalculateGrid {
+    pub fn from(cell_size_in: f32, entities_to_ignore: HashSet<Entity>) -> Self {
```

Comment:
`from` is a bit of an ambiguous function name in Rust, since it can send the message that it is a function from the `From<T>` trait. I would recommend renaming it to `new`

Reply:
Or a possibly cleaner alternative could be to implement `Default` for the structure with sensible default values, then users can just instantiate it by doing:

```rust
let grid = CalculateGrid {
  cell_size,
  ignore,
  ..default()
};
```